python/GafferSceneUI/_SceneViewInspector.py

```diff
@@ -351,14 +355,14 @@ def __attributeLabel( self, attribute ) :
 		name = " ".join( [ IECore.CamelCase.toSpaced( n ) for n in name.split( ":" ) ] )
 		return "{} {}".format( prefix, name )
 
-# EditType defines discreet parameter edit types. The inspector UI assumes that all
+# EditType defines discrete parameter edit types. The inspector UI assumes that all
 # locations with the same edit type can be edited simultaneously using a single
 # PlugValueWidget set to hold all applicable plugs.
 _EditType = IECore.Enum.create( "Source", "Tweak" )
 
-# Defines a (potential) edit for a single location's.
+# Defines a (potential) edit for a single location.
 #  - acquireEdit [callable] will return the plug that performs the edit, creating it if necessary.
-#  - editType [_EditType] the type of edit that (will) be facilitated by the plug.
+#  - editType [_EditType] the type of edit (or potential edit if one doesn't exit yet) facilitated by the plug.
```

Comment:
exit yet -> exist yet

Reply:
I don't know what is wrong with my brain, sorry, fixed [here](https://github.com/GafferHQ/gaffer/pull/3832/commits/540625f6b4452ad4f3dec615e43566cadf0b2bf8).